org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/InfoListSubModel.java

```diff
@@ -98,7 +98,7 @@ public ListModel<ValueNamePair> getSubModel(Object value, int nRows) {
 				.append(nRows);
 			if (lookup instanceof MLookup) {
 				MLookup mlookup = (MLookup) lookup;
-				List<String> displayColumns = mlookup.getLookupInfo().lookupDisplayColumnNames;
+				List<String> displayColumns = mlookup.getLookupInfo().lookupDisplayColumns;
```

Comment:
this break the fix from IDEMPIERE-5588

Reply:
I reverted this change - I did several tests and didn't find why I changed that, tested IDEMPIERE-5588 with UU autocomplete search editor and it's working fine.